src/test/java/com/cloudbees/jenkins/plugins/advisor/ReminderTest.java

```diff
@@ -79,7 +79,7 @@ public void testPluginDisabled() throws Exception {
 
   private void submitForm(WebClient wc, String part, String userEmail, boolean nagOff, boolean acceptTerms) throws Exception {
     HtmlForm form = wc.goTo(part).getFirstByXPath("//form[@action='configure']");
-    form.getInputByName("_.email").setValueAttribute(userEmail);
+    form.getInputByName("_.email").setValue(userEmail);
```

Comment:
Reverting failing change as per comment suggestion.

```suggestion
    form.getInputByName("_.email").setValueAttribute(userEmail);
```